fitbit-react/src/Pages/Homepage/Homepage.js

```diff
@@ -0,0 +1,14 @@
+import React from "react";
+import Footer from "../../Components/Footer/Footer";
+import Navbar from "../../Components/Navbar/Navbar";
```

Comment:
changes: you can import all components to index.js file and import from that file, so you can reduce the lines of import.